src/components/table/column-picker/column-picker.component.html

```diff
@@ -40,7 +71,7 @@
             <ux-icon name="chevron-left"></ux-icon>
         </button>
 
-        <button class="btn button-secondary btn-block" [disabled]="deselected.length === 0" (click)="addAllColumns()">
+        <button class="btn button-secondary btn-block" [disabled]="hasDeselectedItems()" (click)="addAllColumns()">
```

Comment:
Is is possible to calculate this whenever selection changes and store the result in a variable? Ideally we want to avoid calling functions from a binding where possible as this can lead to performance problems